lib/shared/cliOptions.js

```diff
@@ -47,12 +47,6 @@ module.exports = {
     desc: chalk.gray(
       'Print the task dependency tree for the loaded gulpfile.'),
   },
-  depth: {
```

Comment:
I'm fine with being verbose on the flag names but I think we just need to alias `depth` because we've previously defined it and removing it would be a breaking change (and I'd have to publish as 2.0.0)